src/node_lttng_provider.h

```diff
@@ -78,7 +78,7 @@ void NODE_GC_START(v8::GCType type,
 void NODE_GC_DONE(v8::GCType type,
                   v8::GCCallbackFlags flags,
                   v8::Isolate* isolate) {
-  const char* typeStr = "Unkown GC Type";
+  const char* typeStr = "Unknown GC Type";
```

Comment:
If this is an officially supported feature, then semver major. Honestly, version 8 is coming up soon, so maybe just make it semver major anyway.